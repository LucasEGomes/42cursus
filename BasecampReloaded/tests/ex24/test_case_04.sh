NO_EXT="${0%.*}"
OUTPUT=${NO_EXT}.output
RESULT=${NO_EXT}.user
EXPECTED=${NO_EXT}.expected
bash setup.sh $NO_EXT $OUTPUT $RESULT
touch $NO_EXT/all
make all -C $NO_EXT/
find $NO_EXT/ -type f \( -name *.o -o -name *.a \) | sort > $RESULT 2>> $OUTPUT
touch $NO_EXT/re
make re -C $NO_EXT/
find $NO_EXT/ -type f \( -name *.o -o -name *.a \) | sort >> $RESULT 2>> $OUTPUT
touch $NO_EXT/clean
make clean -C $NO_EXT/
find $NO_EXT/ -type f \( -name *.o -o -name *.a \) | sort >> $RESULT 2>> $OUTPUT
touch $NO_EXT/fclean
make re -C $NO_EXT/
make fclean -C $NO_EXT/
find $NO_EXT/ -type f \( -name *.o -o -name *.a \) | sort >> $RESULT 2>> $OUTPUT
diff $EXPECTED $RESULT >> $OUTPUT 2>> $OUTPUT
bash tear_down.sh $NO_EXT
