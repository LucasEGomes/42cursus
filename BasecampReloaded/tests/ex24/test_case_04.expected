test_case_04/ft_putchar.o
test_case_04/ft_putstr.o
test_case_04/ft_strcmp.o
test_case_04/ft_strlen.o
test_case_04/ft_swap.o
test_case_04/libft.a
test_case_04/ft_putchar.o
test_case_04/ft_putstr.o
test_case_04/ft_strcmp.o
test_case_04/ft_strlen.o
test_case_04/ft_swap.o
test_case_04/libft.a
test_case_04/libft.a
